playkit/src/main/java/com/kaltura/playkit/Player.java

```diff
@@ -117,6 +123,11 @@
      */
     void onApplicationResumed();
 
+    /**
+     * Player lifecycle method. Must be called when Activity onConfigurationChanged(Configuration newConfig) called.
+     */
+    void onConfigurationChanged();
```

Comment:
I will change to onOrientationChanged. Those because native Android onConfigurationChanged might caused from different reasons like keyboard input settings changed and etc (those calls not suppose to have any effect on vrPlayer state, because player size is obtained from context that is passed to). So onOrientationChanged sounds more descriptive for this use-case